haskell-org-committee.markdown

```diff
@@ -44,10 +44,18 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
 
-At the end of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
+In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
```

Comment:
Does this render correctly? I would expect `year - or` and `over - the`

Reply:
The text is using the em-dash character (—) and not a hypen (-). Since different dashes only differ in width, distinguishing them can be a real pain in a monospace font :/.

Using an em-dash with no spaces around it is the most common style I've seen in professionally typeset text, although some style guides suggest using an en-dash (–) with spaces around it instead. Using a hyphen with spaces is pretty outdated and only makes sense if you're limited to ASCII—which shouldn't be the case in this day and age!

And if you're as picky as TeX, you'd even use a fourth kind of dash for minus signs :).